trin-execution/Cargo.toml

```diff
@@ -12,11 +12,9 @@ authors = ["https://github.com/ethereum/trin/graphs/contributors"]
 
 
 [dependencies]
-alloy-consensus.workspace = true
-alloy-eips.workspace = true
-alloy-primitives.workspace = true
+alloy = { workspace = true, features = ["eips", "rpc-types-engine", "serde"] }
 alloy-rlp.workspace = true
-alloy-rpc-types = { workspace = true, features = ["engine"]}
+alloy-rpc-types = "0.4.2"
```

Comment:
Can this line be deleted, by adding the `rpc-types` feature in the alloy dependency above?

Reply:
Sadly, no.
The "rpc-types-engine" feature already enables "rpc-types", so adding it wouldn't solve the problem. The problem is that these features don't enable "serde" feature on "alloy-rpc-types" / "alloy-rpc-types-eth" crates.